framework/resources/Functional/filter/pushdown/data/drill_6118_filter_push_down_with_view.sql

```diff
@@ -0,0 +1 @@
+select * from dfs.drillTestDir.`DRILL_6118_parquet_partitioned_by_folders_view` where c1>2;
```

Comment:
Without order by, will result set always the same? For small dataset, it should. But for large dataset, it might not be.

Reply:
Order is not essential for these tests. As I know, Drill Test Framework ignores it unless the query contain "order by" clause. So the tests will fail only if the queries will return wrong data. I verified this case, intentionally changing result order, and the tests passed.